src/main/java/com/badfic/philbot/listeners/Phil.java

```diff
@@ -113,9 +114,11 @@ public static class MessageListener extends ListenerAdapter {
         private final DiscordWebhookSendService discordWebhookSendService;
         private final SwampyGamesConfigDal swampyGamesConfigDal;
         private final BaseConfig baseConfig;
-        private final GitProperties gitProperties;
         private final RestTemplate restTemplate;
 
+        @Setter(onMethod_ = {@Autowired(required = false)})
```

Comment:
When developing locally sometimes the git.properties file isn't generated or it gets deleted. Way easier to just leave this as optional so that you can quickly start up the app instead of having to re-run a mvn command just to generate the git.properties file